data/flaggedProposals.json

```
@@ -8,5 +8,6 @@
   "0xdd85b2ffc6757ecf55f63340651843d1ea872608217f105f12d4328159ac21ea",
   "0x07db27a2237085195e2327ba3bc0e70ece0007217073f67cd0d5686e9b35cd54",
   "0xae5784ab2288b54d52b35e1807f971c3d7569522c13607bba4b67c5a992a42c2",
-  "0x462513291396d7a11e357b8147fd3c140d45b03ffcfb291979b4d5f4e64ec594"
+  "0x462513291396d7a11e357b8147fd3c140d45b03ffcfb291979b4d5f4e64ec594",
+  "0xcab2ec009069d7d49fcd2d98772f5f93de6c88fce49a80126939599e25f51688
```

A quote is missing :D
```suggestion
  "0xcab2ec009069d7d49fcd2d98772f5f93de6c88fce49a80126939599e25f51688"
```